### PR TITLE
since Yamp is not maintained anymore, i switched it for Mages

### PR DESCRIPTION
**Pull request by shufflecode**
Since Yamp is not maintained anymore, I switched it for [Mages](https://github.com/FlorianRappl/Mages)
fixes also #1022

### CODE REVIEW

File: Plugins/Wox.Plugin.Calculator/Languages/en.xaml [hunk 1]
```diff
@@ -4,5 +4,7 @@
 
     <system:String x:Key="wox_plugin_caculator_plugin_name">Calculator</system:String>
     <system:String x:Key="wox_plugin_caculator_plugin_description">Allows to do mathematical calculations.(Try 5*3-2 in Wox)</system:String>
-    
+    <system:String x:Key="wox_plugin_calculator_not_a_number">Not a Number (NaN)</system:String>
+    <system:String x:Key="wox_plugin_calculator_expression_not_complete">Expression wrong or incomplete (Did you forget some Brackets?)</system:String>
```
Comment (jhdxr) [inline Plugins/Wox.Plugin.Calculator/Languages/en.xaml]: I think you mean `Parentheses ()`  rather than `Brackets []`?

**Review by jhdxr — CHANGES_REQUESTED** [review]

File: Plugins/Wox.Plugin.Calculator/Main.cs [hunk 2]
```diff
@@ -38,33 +35,38 @@ public List<Result> Query(Query query)
...
-                                {
-                                    MessageBox.Show("Copy failed, please try later");
-                                    return false;
-                                }
+                                MessageBox.Show("Copy failed, please try later");
+                                return false;
                             }
                         }
-                    };
+                    } };
```
Comment (jhdxr) [inline Plugins/Wox.Plugin.Calculator/Main.cs]: you may have to format this file to get a good code style.

**Review by jhdxr — CHANGES_REQUESTED** [review]

File: Plugins/Wox.Plugin.Calculator/Languages/en.xaml (hunk 1 above)
Comment (shufflecode) [reply Plugins/Wox.Plugin.Calculator/Languages/en.xaml]: Jup, sure thanks for looking over it !

File: Plugins/Wox.Plugin.Calculator/Main.cs (hunk 2 above)
Comment (shufflecode) [reply Plugins/Wox.Plugin.Calculator/Main.cs]: I will change that, thanks for pointing that out.

File: Wox.Infrastructure/Logger/Log.cs [hunk 3]
```diff
@@ -72,7 +72,7 @@ public static void Error(string message)
         public static void Exception(string message, System.Exception e)
         {
 #if DEBUG
-            throw e;
+            //throw e;
```
Comment (jhdxr) [inline Wox.Infrastructure/Logger/Log.cs]: why?

**Review by jhdxr — CHANGES_REQUESTED** [review]

**Comment by jhdxr** [comment]
@happlebao @lances101 Could you take a look at this PR? I've tested it on Win 10.15063 x64, and it works good.

File: Wox.Infrastructure/Logger/Log.cs (hunk 3 above)
Comment (shufflecode) [reply Wox.Infrastructure/Logger/Log.cs]: lack of necessary diligence :/

while debugging i got this exception: 
    
>0x80070005 - Unspecified error
>'Microsoft Corporation' violates pattern constraint of '\bms-resource:.{1,256}'.

so i commented out, tested and forgot to re-enable it

**Review by jhdxr — APPROVED** [review]

**Comment by bao-qian** [comment]
@jhdxr 
you have been invited to the team. then you can approve these request / close issue yourself.

**Comment by jhdxr** [comment]
@happlebao I have accepted the invitation, but it seems I still don't have the permission. Github says 
>Only those with [write access](https://help.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

**Comment by bao-qian** [comment]
@jhdxr 
permission should be granted, check again. let me know if you need anything else.

**Pull request merged**